.final_builds/packages/golang-1.23-windows/index.yml

```diff
@@ -1,12 +1,8 @@
 builds:
   03d4b9d66556f7eeb02cfd48285a40dc33cd3258a921334e71727950d78ed13c:
     version: 03d4b9d66556f7eeb02cfd48285a40dc33cd3258a921334e71727950d78ed13c
-    blobstore_id: babed125-bfe1-475d-7f0f-dd1016ba7df3
+    blobstore_id: 0179c598-a538-47db-7a7f-81c57c3c5429
```

Comment:
changes to all the .final_builds/packages concern me here. We should keep whatever's on main rather than the protobuf branch, right?